### DISCUSSION TITLE
Installing Tailwind CSS with Vanilla JS

### DISCUSSION THREAD

**Discussion by mmikhan** [crxjs/chrome-extension-tools#895] [Q&A]
👋 I thought installing Tailwind CSS with this project would be easy but couldn't figure it out. I have tried following the [PostCSS guide from Tailwind CSS doc](https://tailwindcss.com/docs/installation/using-postcss) but that didn't help.

What do I do after installing TailwindCSS? Do need to make any changes in the Vite config?

Any hints would be appreciated!

**Comment by Toumash** [comment]
Here's the HMR part for the tailwing using crxjs so people are using it
https://github.com/crxjs/chrome-extension-tools/issues/671#issuecomment-2119178416

From the snippets it looks like it doesnt need any config but bydefault the HMR doesnt work. Is it true?

What exectly does not work? In what context - page, content, script, popup?

**Reply by mmikhan** [reply]
I was able to resolve this. Consider this as done!

**Comment by geanrt** [comment]
@Toumash Dude, I have the solution here and have been using it for years to use Tailwind CSS in Crxjs. But there must be a better way to do this (I'm sure). If you need it, let me know and I'll send you the script here.

**Reply by Toumash** [reply]
Does it work with hmr? Share it, please!
